src/Page-Components/Article.jsx

```diff
@@ -31,19 +38,31 @@ const Article = () => {
         <article className="singleArticle">
           <h2 className="text-black font-bold text-xl my-3">{article.title}</h2>
           <img id='singleArticleImg' src={article.article_img_url} alt="" />
+          
           <section id="articleInfo">
             <p className='text-black font-bold'>Topic: <span className='text-black font-medium'>{article.topic}</span></p>
             <p className='text-black font-bold '>Author: <span className='text-black font-medium'>{article.author}</span></p>
             <p className='text-black font-bold'>Posted: <span className='text-black font-medium'>{datePosted}</span></p>
             <p className="text-black my-3">{article.body}</p>
           </section>
-          <button>
-            <img src="src/assets/Like.png" alt="Thumbs Up; like Button" className='h-11'/>
-            <p className='text-black font-bold'>Likes:{article.votes}</p>
-          </button>
+
           <Link to={`/all-articles`}>
               <button type="button" className="px-6 py-3.5 text-base font-bold text-white bg-button-red hover:bg-button-red-hover rounded-lg text-center hover:shadow-xl">Back To Articles</button>
           </Link> 
+
+          <div id="commentCardContainer">
+            <h3 className="text-black font-bold text-lg py-3">Comments</h3>
+            <ul>
```

Comment:
Would be good to have some message here if there are no comments 
You can use article 37 to test this